editor/editor_data.h

```diff
@@ -166,7 +168,7 @@ class EditorData {
 	int get_editor_plugin_count() const;
 	EditorPlugin *get_editor_plugin(int p_idx);
 
-	UndoRedo &get_undo_redo();
+	Ref<EditorUndoRedoManager> get_undo_redo();
```

Comment:
```suggestion
	Ref<EditorUndoRedoManager> &get_undo_redo();
```

I think this can also be set to return a reference in order to avoid increasing & decreasing the refcounter for temporary objects.

Callers can still retain the reference by assigning the return value to a `Ref<T>` instead of `Ref<T> &` if they want.